odl/contrib/README.md

```diff
@@ -0,0 +1,98 @@
+# The `odl.contrib` sub-package
+
+This is a special place for user-contributed modules. Its intention is to provide a "fast lane" for you as a contributor to include code that is
+
+- outside the main ODL library,
+- experimental and untested,
+- still subject to major breaking changes or
+- "not quite there yet" in terms of documentation, testing or completeness.
+
+The general idea is that you get your code quickly merged without *having to* meet the quality and consistency standards of main ODL that we ensure through extended code reviews. The reasons why we still want your contributed code to be part of the main ODL *repository* are (1) that ODL is intended to follow a ["batteries included"](https://www.python.org/dev/peps/pep-0206/#batteries-included-philosophy) philosophy, and (2) to give you as a contributor maximum visibility.
+
+**Important:** Responsibility for code that you contribute will remain with you. This means in particular that issues on the tracker about your module will be assigned to you. Of course, others will try to help if necessary.
+
+**Note:** If you consider your code to be useful for the main library, please consider filing a pull request anyway. We try to be as helpful as possible in our reviews and make suggestions, so that getting the code ready for merging doesn't take too long.
+Note also that contributed modules that attract significant attention and user adoption will be considered for core ODL.
+
+Here are some guidelines for contributors regarding the scope and the rules of this platform.
+
+
+## Scope
+
```

Comment:
Feels very empty to start with a list, perhaps add something like:

"To determine if your code is suited for `odl.contrib`, consider the following guidelines:"